src/components/Login/FindPassword.js

```diff
@@ -34,7 +34,7 @@ class FindPassStep extends Component {
             placeholder={this.props.inputHint}
             onChangeText={onChangeText}
             placeholderTextColor="#d8d8d8"
-            underlineColorAndroid="#efeff2"
+            underlineColorAndroid="rgba(255, 255, 255, 0)"
```

Comment:
I think it's necessary to have a conversation about it.
If we think @ifndefdeadmau5 's opinion is resonable, we might have lots of changes.